dev-python/pipenv/pipenv-2021.5.29-r1.ebuild

```diff
@@ -20,11 +20,17 @@ KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE="test"
 RESTRICT="!test? ( test )"
 
-PATCHES="${FILESDIR}/${PN}-${PV//./-}-${PR}-remove-attr-vendor-import.patch"
+PATCHES=( 
+	"${FILESDIR}/${PN}-${PV//./-}-${PR}-remove-attr-vendor-import.patch"
+	"${FILESDIR}/${PN}-${PV//./-}-${PR}-remove-colorama-vendor-import.patch"
+	)
 
 RDEPEND="
 	${PYTHON_DEPS}
+	dev-python/attr[${PYTHON_USEDEP}]
 	dev-python/certifi[${PYTHON_USEDEP}]
+	>=dev-python/colorama-0.4.4[${PYTHON_USEDEP}]
+	>=dev-python/jinja-3.0.1[${PYTHON_USEDEP}]
```

Comment:
Adding new RDEPEND requires a revbump.

Reply:
OK, will change this, to be a new version.